### ISSUE TITLE
Softmax not suitable for normalizing the amplitudes

### ISSUE THREAD

**Issue by linfeng-du** [thuml/Time-Series-Library#202]
Since the amplitudes are relatively large (could be tens to one hundred), softmax may not be suitable for normalizing them. Currently it's behaving pretty much like a max function, meaning that only the representations from the top 1 frequency is effectively leveraged. For others frequencies, the normalized weights are negligibly small.

Have the authors experimented with different aggregating methods (e.g. convolution as in MICN) to show that incorporating multiple resolutions  could actually be beneficial?

**Comment by wuhaixu2016** [comment]
Hi, thanks for your great question and suggestion.
(1) In our experiments, the amplitudes are not that large. Softmax is necessary.
(2) You may find the hyper-parameter sensitivity analysis in Appendix B. More kernels will bring better results, especially for M4 dataset.
